unattended-upgrade

```diff
@@ -978,9 +1002,16 @@ def _send_mail_using_sendmail(from_address, to_address, subject, body):
     return ret
 
 
-def send_summary_mail(pkgs, res, pkgs_kept_back, pkgs_removed,
-                      pkgs_kept_installed, mem_log, dpkg_log_content):
-    # type: (str, bool, List[str], List[str], List[str], StringIO, str) -> None
+def send_summary_mail(pkgs,                 # type: List[str]
```

Comment:
Probably not for this PR but I wonder if send_summary_mail should simply take the new UnattendedUpgradeResult class?

Reply:
@mvo5 I had the same thinking but send_summary_mail() does not consume update_stamp thus I just left it this way. Also that should be a separate commit which can come later.